apps/storybook/src/LineVis.stories.tsx

```diff
@@ -70,25 +82,31 @@ DescendingAbscissas.args = {
 };
 
 export const ErrorBars = Template.bind({});
-
 ErrorBars.args = {
   dataArray,
-  domain: [-31, 31], // Extend domain to fit error bars
   errorsArray,
   showErrors: true,
 };
 
 export const AuxiliaryArrays = Template.bind({});
-
 AuxiliaryArrays.args = {
   dataArray: primaryArray,
   auxArrays: [secondaryArray, tertiaryArray],
-  domain: combinedDomain,
 };
 
+export const TypedArrays = Template.bind({});
+TypedArrays.args = {
+  dataArray: toTypedNdArray(primaryArray),
+  errorsArray: toTypedNdArray(errorsArray),
+  auxArrays: [toTypedNdArray(secondaryArray)],
+  showErrors: true,
+};
```

Comment:
The new story that checks that typed arrays work as expected:

![image](https://user-images.githubusercontent.com/2936402/152558830-fa7bc266-f032-46fd-80e9-43a52b58540c.png)